packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -193,6 +195,11 @@ class CupertinoSlider extends StatefulWidget {
   /// Defaults to the [CupertinoTheme]'s primary color if null.
   final Color activeColor;
 
+  /// The color to use for the thumb of the slider.
+  ///
+  /// Defaults to [CupertinoColors.white].
```

Comment:
Could you also document the `assert(thumbColor != null)` constraint?

Reply:
Is `must not be null` ok? I saw that in other namespaces
```dart
  /// The color to use for the thumb of the slider.
  ///
  /// Thumb color must not be null.
  ///
  /// Defaults to [CupertinoColors.white].
```

Reply:
Or do you think it would be better to tell in the documentation of the constructor that you can give the thumb a color, but it mustn't be null? (I saw that on other namespaces too, but I thought it would be weird to tell in the constructor that you can change the thumb color without mentioning you can change the active color.

Reply:
Ah sorry I wasn't being clear. "Thumb color must not be null." sounds good.